tests/integration/bills.test.js

```diff
@@ -225,6 +225,70 @@ describe('BILL ROUTES - POST /bills', () => {
         },
       ],
     },
+    {
```

Comment:
je pense que ce serait mieux de faire un deuxieme test pour ca : 
le 1er "should create new internal bill"
et le 2eme "should create new external bill"
un truc comme ca